packages/open-next/src/wrappers/node.ts

```diff
@@ -10,6 +10,7 @@ const wrapper: WrapperHandler = async (handler, converter) => {
     const internalEvent = await converter.convertFrom(req);
     const _res: StreamCreator = {
       writeHeaders: (prelude) => {
+        res.setHeader("Set-Cookie", prelude.cookies);
```

Comment:
I don't think `Set-Cookie` allows for setting multiple cookies at once (could be wrong). Need to split the cookies and `res.addHeader` if so.

Reply:
Actually it does ( I thought the same thing as well at first ).

Reply:
That's really weird, according to the spec: https://httpwg.org/specs/rfc6265.html#overview it shouldn't. Since the comma character may also be part of the cookie value.

Reply:
`prelude.cookies` is an array, it's from here 
https://github.com/sst/open-next/blob/faefb33a2a485c571c2cc43680f1b702a03b58c7/packages/open-next/src/http/openNextResponse.ts#L187

Reply:
It will work with an array.

Reply:
Actually there was an issue here ( i forgot to remove the custom wrapper on my test :roll_eyes: ).
We needed to remove the `set-cookie` header from the headers, otherwise the broken one would take precedence.
There was also another issue with the set-cookie that affected everything. If cookies were set both in the middleware and in a route, only the middleware ones would be applied